Profile page

liamlenihan opened the pull request:
I have uploaded mng_content.php, myprofile.php and userupadte.php as these are the files which handle the userprofile and update routines. All feedback and changes welcome as they will allow me to further my skills within the TEN development team and to improve the coding and user interface aspects and as such the entire user experience.

Testing screenshots illustrate the profile pages functionailty as demonstrated below:- 
![image](https://cloud.githubusercontent.com/assets/26595653/24958838/adf02d94-1f88-11e7-8813-594ab0bd02f1.png)
![image](https://cloud.githubusercontent.com/assets/26595653/24958854/b764deba-1f88-11e7-8f9e-d50ae8974fcd.png)
![image](https://cloud.githubusercontent.com/assets/26595653/24958890/d670ce36-1f88-11e7-921d-b713e8cba6f0.png)
![image](https://cloud.githubusercontent.com/assets/26595653/24958899/e0a3f05e-1f88-11e7-8835-00380965ff06.png)

liamlenihan committed:
Add files via upload

I have uploaded mng_content.php, myprofile.php and userupadte.php as these are the files which handle the userprofile and update routines. All feedback and changes welcome.

liamlenihan committed:
nav.inc.php

nav.inc.php file to ensure all menu items are displyed.

liamlenihan committed:
Placeholder.jpg 

Placeholder.jpg has been uploaded for the rectification of issues encountered during the deployment phase.

liamlenihan committed:
Add files via upload

Re uploaded to the correct folder for example, images.

liamlenihan committed:
Delete placeholder.jpg

liamlenihan committed:
Add files via upload

liamlenihan committed:
Delete placeholder.jpg

amigafan2003 committed:
Converted table to divs to fix responsive issues

amigafan2003 committed:
Added appropriate avatar placeholder image

amigafan2003 committed:
Placeholder code expects jpg not png

amigafan2003 reviewed (changes requested):
Hi Liam.  I have proposed some changes for this functionality - please see amended code and discussion on discord (mainly around responsiveness of tables etc).  Overall the functionality is looking good but it looks like there may be a bug with the image upload function - it just hangs at mng_user.php after submitting an image.  If you fix these issues, we should be good to go :-)

liamlenihan committed:
mng_user.php

I have had a really good attempt at the mng_user.php file; however, their is still some work to be completed as it is a core requirement of the Tailor Edited News (TEN) web application.

liamlenihan committed:
Add files via upload
